.github/workflows/tests.yml

```diff
@@ -45,5 +45,5 @@ jobs:
       - name: Integration Tests
         run: |
           cd examples/
```

Comment:
Shouldn't we use `cd examples/intro/` here?

Reply:
And in `docker-compose.yml` also we should change the context from `../` to `../../`